@navikt/ds-react/package.json

```
@@ -28,14 +27,14 @@
     "/src"
   ],
   "scripts": {
-    "cjs-package": "copyfiles -f config/package.json cjs",
+    "modify-package": "node ../../utilities/scripts/packageHandler.js",
```

Vanskelig å finne gode, beskrivende navn her.

```suggestion
    "package-json-pointers-to-esm": "node ../../utilities/scripts/packageHandler.js",
```